runcon: add page (#3297)
tldr-pages/tldr: pages/linux/runcon.md

# runcon

> Run a program in a different SELinux security context.
> With neither CONTEXT nor COMMAND, print the current security context.

> - Determine current domain:

Reviewer: Perhaps this will read better?
```suggestion
- Determine the current domain:
```

Author: Sounds good, changes made!

`runcon`

> - Specify the domain that you want to run a program or script in:

Reviewer: Perhaps this will read better?
```suggestion
- Specify the domain to run a command in:
```

Author: Sounds good, changes made!

> `runcon -t CONTEXT-TYPE_t {{command}}`

Reviewer 2: This has a - in between CONTEXT and TYPE, but the later examples use _. Which is it ?

Also, don't these need to be inside tokens (`{{}}`)

> - Specify the role type that you want to run a program or script with:

Reviewer: Perhaps this will read better?
```suggestion
- Specify the role type to run a command with:
```

Author: Sounds good, changes made!

`runcon -r ROLE_TYPE_r {{command}}`

> - Specify the full context to run a program or script with:

Reviewer: Perhaps this will read better?
```suggestion
- Specify the full context to run a command with:
```

Author: Sounds good, changes made!
